src/psyclone/tests/psyir/frontend/fparser2_literals_test.py

```diff
@@ -148,6 +209,8 @@ def test_handling_literal_precision_2(value, dprecision, intrinsic):
     assert literal.datatype.intrinsic == intrinsic
     if intrinsic == ScalarType.Intrinsic.BOOLEAN:
         assert ".{0}.".format(literal.value) == value.lower()
+    elif intrinsic == ScalarType.Intrinsic.CHARACTER:
+        assert "'{0}'".format(literal.value) == value
```

Comment:
Again, perhaps mix the case of the original string.

Reply:
Done.